platform/darwin/src/MGLGeoJSONSource.h

```diff
@@ -59,6 +59,15 @@ extern const MGLGeoJSONSourceOption MGLGeoJSONSourceOptionSimplificationToleranc
 /**
  A GeoJSON source.
 
+ ### Example ###
+ 
+ ```swift
+ var coordinates: [CLLocationCoordinate2D] = []
+ let polyline = MGLPolylineFeature(coordinates: &coordinates, count: UInt(coordinates.count))
+ let source = MGLGeoJSONSource(identifier: "lines", features: [polyline], options: nil)
+ self.mapView.style().add(source)
```

Comment:
Nit: other examples omit `self.`, presumably because `mapView` could be a local variable.